src/core/ddsi/src/q_plist.c

```diff
@@ -2763,6 +2849,29 @@ void nn_log_xqos (uint32_t cat, const struct ddsrt_log_cfg *logcfg, const dds_qo
   });
   DO (PRISMTECH_ENTITY_FACTORY, { LOGB1 ("entity_factory=%u", xqos->entity_factory.autoenable_created_entities); });
   DO (CYCLONE_IGNORELOCAL, { LOGB1 ("ignorelocal=%u", xqos->ignorelocal.value); });
+  DO (PROPERTY_LIST, {
```

Comment:
You know, I don't mind the addition of the property list here so much as that I mind my own laziness in not rewriting it when I added those tables ... most of all this special-purpose printing code could just be done using the same tables as the serialisation, comparison, &c.

It would be especially nice if it would print into a string buffer rather than using the tracing macros, because then the implementation of printing the content of parameter-list samples would become trivial. And the current printing of QoS values could just go away.

I don't think it reasonable to require all those changes in this PR, but if you were to decide to do it anyway, it would be appreciated!

Reply:
There's a similar PR in the works that add some msg parameters to plist (iso xqos). Maybe I'll do it in that PR. 

@eboasson, question though.
Currently, strings like "ignorelocal=", "lifespan=", etc are added to the log.
Don't you lose that kind of information when using the tables?